src/components/Profile/components/SchedulePanel/components/ScheduleCalendar/index.jsx

```diff
@@ -1,29 +1,19 @@
+import React, { useEffect, useState } from 'react';
 import GordonLoader from 'components/Loader';
 import Moment from 'moment';
-import { Component, Fragment } from 'react';
 import { Calendar, momentLocalizer } from 'react-big-calendar';
-import myscheduleService from 'services/myschedule';
 import scheduleService from 'services/schedule';
 import session from 'services/session';
-// @TODO CSSMODULES - Schedule Calendar needs work but left as normal for now
 import './ScheduleCalendar.css';
 
-export default class GordonScheduleCalendar extends Component {
-  constructor(props) {
-    super(props);
+const GordonScheduleCalendar = (props) => {
+  const [loading, setLoading] = useState(true);
+  const [selectedEvent, setSelectedEvent] = useState(null);
+  const [isDoubleClick, setIsDoubleClick] = useState(false);
+  const [currentSession, setCurrentSession] = useState([]);
+  const [eventInfo, setEventInfo] = useState([]);
 
-    this.state = {
-      loading: true,
-      myScheduleOpen: false,
-      disabled: true,
-      selectedEvent: null,
-      isDoubleClick: false,
-      currentSession: [],
-    };
-    this.eventInfo = [];
-  }
-
-  customEventPropGetter = (event, start, end, isSelected) => {
+  const customEventPropGetter = (event, start, end, isSelected) => {
     if (event.id > 1000) {
```

Comment:
I know you didn't change this line, but it needs a comment to explain the meaning of the magic number 1000.  

I wonder if this function is even needed any more.  Can it be deleted?

Reply:
the line `if (event.id > 1000) {` was used in both the schedulePanel file and the scheduleCalendar file. The usage in both files became irrelevant because we no longer have custom schedules or users being able to edit their schedule by double clicking (because official course schedules cannot be edited). I removed this from both files.